src/jit/codegen.h

```diff
@@ -465,11 +465,11 @@ protected :
                                     IL_OFFSETX                                    ilOffset);
 
 
+#if defined(_TARGET_ARM_)
     //
```

Comment:
I like Bruce's suggestion, but I might use something like CLANG_FORMAT_COMMENT_ANCHOR to make it easier to tell why it's there without having to look it up.

---
In reply to: [72640280](https://github.com/dotnet/coreclr/pull/6491#discussion_r72640280) [](ancestors = 72640280)